lib/bundler_api/cdn.rb

```diff
@@ -7,6 +7,7 @@ def self.purge_specs(client = self.client)
     client.purge_path '/latest_specs.4.8.gz'
     client.purge_path '/specs.4.8.gz'
     client.purge_path '/prerelease_specs.4.8.gz'
+    client.purge_path "/names"
```

Comment:
We also need to expire the /versions cached in fastly whenever there is a new gem version, so I think that means here as well?